src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java

```diff
@@ -74,10 +74,10 @@ public void testMessageReceived() {
     // Stubbing
     when(mockedMessage.getLabel()).thenReturn("Fire Event");
 
-    // Execution    
```

Comment:
As a side note, do we have a "no spaces after comments" in our checkstyle?  It should not be happening, and I'm seeing it in a lot of places.